meteor-app/imports/startup/server/email.js

```diff
@@ -33,5 +37,96 @@ Meteor.methods({
     } catch (error) {
       log.error('Error from email gateway', error)
     }
+  },
+  sendMembershipEmail(to, name, type, expiry, apiKey) {
+    sgMail.setApiKey(Meteor.settings.private.sendgridApikey)
+    const options = {
+      to,
+      from: Meteor.settings.private.fromEmail,
+      templateId: apiKey,
+      dynamic_template_data: {
+        name,
+        type,
+        expiry
+      }
+    }
+    sgMail.send(options)
+  },
+
+  sendMembershipRenewals() {
+    Members.find({}).forEach(member => {
+      Purchases.find({
+        memberId: member._id,
+        code: '/PA-MEMB/',
+        expiry: { $lt: new Date() }
+      }).forEach(purchase => {
+        Meteor.call(
+          'sendMembershipEmail',
+          member.email,
+          member.name,
+          purchase.productName,
+          Moment(purchase.expiry).format('Do MMM YYYY'),
+          Meteor.settings.private.expiredMembershipID
+        )
+      })
+    })
+  },
+  sendMembershipReminderEmail() {
+    Members.find({}).forEach(member => {
+      Purchases.find({
+        memberId: member._id,
+        $or: [{ code: 'PA-MEMB-12' }, { code: 'PA-MEMB-3' }],
```

Comment:
Shouldn't this use the regular expression as above?

Reply:
Actually I did but i think it has changed when I pulled 'shop' branch. I will change it again.